src/common/menuService.js

```diff
@@ -0,0 +1,48 @@
+let currentMenu;
+let nextMenu;
+const menu = [];
+const dayNames = {
+    mon: 'Monday',
+    tue: 'Tuesday',
+    wed: 'Wednesday',
+    thu: 'Thursday',
+    fri: 'Friday',
+    sat: 'Saturday',
+}
+function setWeekMenu(obj, flag) {
```

Comment:
instead of flag name the second argument in a more meaningful way. What does this flag do?